pandas/parser.pyx

```diff
@@ -42,6 +42,9 @@ import sys
 
 cdef bint PY3 = (sys.version_info[0] >= 3)
 
+cdef int QUOTE_NONE = 3  # csv.QUOTE_NONE
+cdef int QUOTE_NONNUMERIC = 2  # csv.QUOTE_NONNUMERIC
```

Comment:
They're already enums in `csv.*` but I wasn't sure whether we should be importing `csv` just for those two enums.